pkg/kubernetes/announcement_handlers.go

```diff
@@ -14,47 +14,54 @@ import (
 )
 
 // PatchSecretHandler patches the envoy bootstrap config secrets based on the PodAdd events
-func PatchSecretHandler(kubeClient kubernetes.Interface) {
+func PatchSecretHandler(kubeClient kubernetes.Interface) chan struct{} {
```

Comment:
Could you update the comment in L16 to reflect what is being returned?

Reply:
Added comment